.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,30 @@
+name: Tag on merge to master
+
+on:
+  push:
+    tags: [ '*' ]
```

Comment:
I think using `'*'` is okay, unless we have any tags with a `/` in them? I think our tags follow `x.x.x` pattern as far as I can tell?

From the [GitHub docs, GitHub Actions Filter pattern cheat sheet](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

> You can use special characters in path, branch, and tag filters.
>
> - `*`: Matches zero or more characters, but does not match the `/` character. For example, `Octo*` matches `Octocat`.
> - `**`: Matches zero or more of any character.

We're correctly quoting the wildcard/filter pattern, according to the docs:

The characters `*`, `[`, and `!` are special characters in YAML. If you start a pattern with `*`, `[`, or `!`, you must enclose the pattern in quotes.

Reply:
Pretty sure this is okay? We're really only interested in automatic tags, which are always `2.1.xxx`, so this should work just fine for that.

Reply:
I think it's OK as well, but just wanted to verify. 😉